kd/deployment.yaml

```
@@ -0,0 +1,243 @@
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - name: certs
+              mountPath: /certs
+            - name: bundle
+              mountPath: /etc/ssl/certs
+              readOnly: true
+          resources:
+            limits:
+              memory: 32Mi
+              cpu: 400m
+            requests:
+              memory: 8Mi
+              cpu: 100m
+
+        - name: proxy
+          image: quay.io/ukhomeofficedigital/nginx-proxy:v3.4.12
+          imagePullPolicy: Always
+          securityContext:
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - SETUID
+                - SETGID
+          env:
+            - name: PROXY_SERVICE_HOST
+              value: '127.0.0.1'
+            - name: PROXY_SERVICE_PORT
+              value: '8080'
+            - name: NAXSI_USE_DEFAULT_RULES
+              value: 'FALSE'
+            - name: ENABLE_UUID_PARAM
+              value: 'FALSE'
+            - name: HTTPS_REDIRECT
+              value: 'FALSE'
+            - name: BASIC_AUTH
+              value: /etc/nginx/authsecrets/htpasswd
+            - name: SERVER_CERT
+              value: /certs/tls.pem
+            - name: SERVER_KEY
+              value: /certs/tls-key.pem
+            - name: ADD_NGINX_SERVER_CFG
+              value: >-
+              location = /reload { allow 127.0.0.1; deny all; content_by_lua_block
+              { os.execute("touch /tmp/nginx-reload-triggered;
+              /usr/local/openresty/nginx/sbin/nginx -s reload;
+              touch /tmp/nginx-reload-complete;") } }
+          volumeMounts:
+            - name: certs
+              mountPath: /certs
+              readOnly: true
+            - name: secrets
+              mountPath: /etc/nginx/secrets
+              readOnly: true
+            - name: auth-secrets
+              mountPath: /etc/nginx/authsecrets
+              readOnly: true
+          ports:
+            - name: https
+              containerPort: 10443
+          resources:
+            limits:
+              memory: 32Mi
+              cpu: 400m
+            requests:
+              memory: 4Mi
+              cpu: 100m
+
+        - name: hocs-case-creator
+          image: quay.io/ukhomeofficedigital/hocs-case-creator:{{.VERSION}}
+          imagePullPolicy: Always
+          securityContext:
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - SETUID
+                - SETGID
+          envFrom:
+            - configMapRef:
+                name: hocs-queue-config
+          env:
+            - name: JAVA_OPTS
+              value: >-
+              -Xms768m -Xmx4096m -XX:+UseG1GC -Djavax.net.ssl.trustStore=/etc/keystore/truststore.jks
+              -Dhttps.proxyHost=hocs-outbound-proxy.{{.KUBE_NAMESPACE}}.svc.cluster.local
+              -Dhttps.proxyPort=31290 -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local
```

This also needs proper indentation:
```suggestion
              value: >-
                -Xms768m -Xmx4096m -XX:+UseG1GC
                -Djavax.net.ssl.trustStore=/etc/keystore/truststore.jks
                -Dhttps.proxyHost=hocs-outbound-proxy.{{.KUBE_NAMESPACE}}.svc.cluster.local
                -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local
                -Dhttps.proxyPort=31290
```